### ISSUE TITLE
Felix OSGi Web Console fails when installed

### ISSUE THREAD

**Issue by blackard** [payara/Payara#1310]
# Description #

Felix OSGi Web Console fails

## Expected Outcome

The OSGi Console is diaplayed and provides a mechanism to examine the state of the OSGi container.

## Current Outcome

HTTP 404 - Not Found error in the tab body after clicking on OSGi Console tab in Admin Server node.

## Steps to reproduce (Only for bug reports) 

1. Download the [Felix OSGi Web Console component for GlassFish](https://maven.java.net/content/repositories/public/org/glassfish/main/packager/glassfish-osgi-gui/4.1.1/) component and extract the contents into the deployment - this adds glassfish-osgi-console-plugin.jar to <payara4>/glassfish/modules as well as felix-webconsole-extensions.jar and org.apache.felix.webconsole.jar to <payara4>/glassfish/modules/autostart.

2. Start (or restart) the domain.

3. Connect to the console at http://localhost:4848/, select __server (Admin Server)__, click the __OSGi Console__ tab.

    1. Alternatively, access the OSGi Web Console directly at http://localhost:8080/osgi/system/console

## Context

The OSGi Console works fine when deployed on a Plain Jane GlassFish 4.1.1 deployment.  In addition to being accessible from the OSGi Console tab on the Server Console, the application is supposed to be accessible at http://localhost:8080/osgi/system/console.

This console provides a convenient mechanism to observe the state of the OSGi container and deploy bundles.

## Environment ##

- **Payara Version**: 4.1.1.164
- **Edition**: Full
- **JDK Version**: Oracle JDK 8 u111
- **Operating System**: Windows 10

**Comment by evilrich** [comment]
I have experienced the same problem when converting from Glassfish 4.1.1 to Payara - not just with the OSGi Web Console – but with any OSGi component that attempts to get `org.osgi.service.http.HttpService`. The Glassfish web container is just not publishing that service any longer.

I have tracked the problem down to [this change](https://github.com/payara/Payara/commit/6968f373f6cee2781b4e6523a2d4dd7e46a366c3#diff-daa8487a616cdd8e90ac2fde14b2b814)

See the [com.sun.enterprise.web.WebModule class](com.sun.enterprise.web). The reference to `webBundleDescriptor` is null leading to an NPE. If I replace `webBundleDescriptor.getServletInitializersEnabled()` with `true`, then the web container successfully publishes the HttpService and everything works. I do not know what the correct fix is, however.
